app/src/main/res/layout/item_event.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think CardView at the root and a ConstraintLayout as the child could be a better approach. that way a LinearLayout won't be needed.

Reply:
I agree

Reply:
ConstraintLayout is crucial at the root. I can eliminate LinearLayout though.